models/orderDao.js

```diff
@@ -0,0 +1,172 @@
+const { database } = require('./dataSource');
+const orderStatusEnums = require('./enums');
+
+
+const createOrder = async ( userId , roomId, checkIn, checkOut, pointChange ) => {
+  const queryRunner = database.createQueryRunner();
+  await queryRunner.connect();
+  await queryRunner.startTransaction();
+  try{
+    const data = await queryRunner.query ( `
+      INSERT INTO orders
+        (user_id,
+          room_id,
+          check_in_date,
+          check_out_date)
+        VALUES
+        (?,?,?,?)
+    `,[ userId, roomId, checkIn, checkOut ])
+
+    await queryRunner.query(`
+      UPDATE users
+        SET point=${pointChange}
+      WHERE id=${userId}
+    `)
```

Comment:
위의 쿼리와 동일하게 배열에 parameter를 담아 매핑하는 코드로 수정해주세요!

다른 쿼리문도 확인 후 수정해주세요~

```suggestion
    await queryRunner.query(`
      UPDATE users
        SET point= ?
      WHERE id= ?
    `, [pointChange, userId])
```